Feature/reg ensemble fix cpreq  change cpreq to cp and check if file exists before cp

BinbinZhou-NOAA opened the pull request:
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

BinbinZhou-NOAA committed:
change cpreq to cp and check if file exists before cp

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/reg_ensemble_fix_cpreq

BinbinZhou-NOAA committed:
change cpreq and check if file exits for narre plot script

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/reg_ensemble_fix_cpreq

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/reg_ensemble_fix_cpreq

BinbinZhou-NOAA committed:
Fix [[ ]] && [[ ]] and add -v to cp

BinbinZhou-NOAA committed:
update sref/href gather scripts

ShelleyMelchior-NOAA commented:
I have cloned your fork and checked out branch `feature/reg_ensemble_fix_cpreq`

BinbinZhou-NOAA commented:
Shelley,

  If you test, pleas test both stats and plots jobs for all 3 regional
ensembles

Thanks!

Binbin

On Thu, Feb 8, 2024 at 3:36 PM Shelley Melchior ***@***.***>
wrote:

> I have cloned your fork and checked out branch
> feature/reg_ensemble_fix_cpreq
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1934894305>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFNJRYI3LZJVDGMPEL3YSUZLHAVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZUHA4TIMZQGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks for the extra instructions, Binbin! I will get to the testing tomorrow and update the PR w/ results.

ShelleyMelchior-NOAA commented:
### SREF
@BinbinZhou-NOAA , I have tested all of the sref stats and plots processes.
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_*stats.o*`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/mesoscale/sref.20240206/`
The stats processes are still running as of this post. precip and grid2obs are each ~35 mins in.
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_*_plots.o*`
COMOUTplots=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20240207`
I see no `cpreq`s and I see many `cp -v`s. 👍

BinbinZhou-NOAA commented:
Shelley,

  I looked at the o* file for sref grid2obs stat job. It seems that the
running was finished, the final stat file
was created in the working directory ($DATA), but it was not copy to the
COMOUT.  Then I further
checked your ush/mesoscale/evs_sref_gather.sh in
/lfs/h2/emc/vpppg/noscrub/shelley.melchior/githubwkspc/forks/BZ/EVS/ush/mesoscale
Line 49:
    if [ $SENDCOM = 'YES' ] && [ -s
${WORK}/gather/${vday}/${modnam}_${verify}_${vday}.stat ] ; then
   echo "cp -v ${WORK}/gather/${vday}/${modnam}_${verify}_${vday}.stat
 $COMOUTfinal/evs.stats.${model}.${verify}.v${vday}.stat">>run_gather_${verify}.sh
    fi
This is the old one and not correct as you pointed out. I have updated it
yesterday in gitgub, as
echo "[[ $SENDCOM = YES  &&  -s
${WORK}/gather/${vday}/${modnam}_${verify}_${vday}.stat ]] &&
cp -v ${WORK}/gather/${vday}/${modnam}_${verify}_${vday}.stat
$COMOUTfinal/evs.stats.${model}.${verify}.v${vday}.stat">>run_gather_${verify}.sh
So please update it in your script and re-run it.

Thanks!

Binbin


  chmod +x run_gather_${verify}.sh

On Fri, Feb 9, 2024 at 10:42 AM Shelley Melchior ***@***.***>
wrote:

> SREF
>
> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , I have tested all
> of the sref stats and plots processes.
> logfiles:
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_*_stats.o*
> COMOUT=
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/mesoscale/sref.20240206/
> The stats processes are still running as of this post. precip and grid2obs
> are each ~35 mins in.
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_*_stats.o*
> COMOUTplots=
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20240207
> I see no cpreqs and I see many cp -vs. 👍
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1936154522>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJ7CUVFVRQ6H3AMYXLYSY7XTAVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZWGE2TINJSGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
You're correct, @BinbinZhou-NOAA ! My apologies. I saw you had pushed the changes but I never pulled them into my clone of your fork. I will do presently and re-run sref grid2obs stats.

In the mean time, I have the href processes running. But I kicked them all off before updating your clone. Please review and if you see problems let me know and I will restart the href processes, too.

### HREF
@BinbinZhou-NOAA , I have tested all of the href stats and plots processes.
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_*.o*`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/cam/href.20240208`
The stats processes are still running as of this post. precip and grid2obs are each ~35 mins in.
COMOUTplots=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/cam/atmos.20240207`
I see no cpreqs and I see many cp -vs. 👍

BinbinZhou-NOAA commented:
Shelley,

  Thanks!,  I am monitoring your href jobs, and noticed the spcoutlook stat
job was finished. Its final stat file is in the
COMOUT directory and matches the spcoutlook file in parallel directory.

Binbin

On Fri, Feb 9, 2024 at 12:05 PM Shelley Melchior ***@***.***>
wrote:

> You're correct, @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> !
> My apologies. I saw you had pushed the changes but I never pulled them into
> my clone of your fork. I will do presently and re-run sref grid2obs stats.
>
> In the mean time, I have the href processes running. But I kicked them all
> off before updating your clone. Please review and if you see problems let
> me know and I will restart the href processes, too.
> HREF
>
> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , I have tested all
> of the href stats and plots processes.
> logfiles: /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_*.o*
> COMOUT=
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/cam/href.20240208
> The stats processes are still running as of this post. precip and grid2obs
> are each ~35 mins in.
> COMOUTplots=
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/cam/atmos.20240207
> I see no cpreqs and I see many cp -vs. 👍
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1936281377>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFN5CJMFCSP5BKKTULTYSZJLVAVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZWGI4DCMZXG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/reg_ensemble_fix_cpreq

ShelleyMelchior-NOAA commented:
Hi @BinbinZhou-NOAA , the re-run for sref grid2obs stats has completed. The final stats file is in COMOUT.
```
$ ll /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/mesoscale/sref.20240207
total 4924
-rw-r--r-- 1 shelley.melchior vpppg 5041910 Feb  9 18:14 evs.stats.sref.grid2obs.v20240207.stat
```

ShelleyMelchior-NOAA commented:
@BinbinZhou-NOAA , while href finished up, I went ahead and started narre.

### NARRE
I have tested all of the narre stats and plots processes.
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_*.o*`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240208`
COMOUTplots=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240208`
I see no `cpreq`s and I see many `cp -v`s. 👍

BinbinZhou-NOAA commented:
Yes, I see it, but the size is a little bit smaller than the parallel file.
I'll look at it to see what is missing.

Binbin

On Fri, Feb 9, 2024 at 1:25 PM Shelley Melchior ***@***.***>
wrote:

> Hi @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , the re-run for
> sref grid2obs stats has completed. The final stats file is in COMOUT.
>
> $ ll /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/mesoscale/sref.20240207
> total 4924
> -rw-r--r-- 1 shelley.melchior vpppg 5041910 Feb  9 18:14 evs.stats.sref.grid2obs.v20240207.stat
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1936406793>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFPWKLQSYV2DNIWEMALYSZSX5AVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZWGQYDMNZZGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Yes, NARRE's stat and plot output files are all good

Binbin

On Fri, Feb 9, 2024 at 1:50 PM Shelley Melchior ***@***.***>
wrote:

> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , while href
> finished up, I went ahead and started narre.
> NARRE
>
> I have tested all of the narre stats and plots processes.
> logfiles: /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_*.o*
> COMOUT=
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240208
> COMOUTplots=
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240208
> I see no cpreqs and I see many cp -vs. 👍
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1936438425>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFPI3FMQJR5T7MTMVC3YSZVVZAVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZWGQZTQNBSGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Shelley,

  I found the problem: the VIS and HGT (ceiling) are missing in the final
stat grid2obs file.
Further debugging: you ram sref grid2obs for VDATE=20240207 only. But it
depends on grid2obs_cnv.
So you should run grid2obs_cnv first. After it is finished, run grid2obs.
Another option: I noticed that you have run  sref grid2obs cnv for
VDATE=20240206. Its output data are
saved in the COMOUTsmall (e.g. atmos.VDATE directory)
So you can set VDATE=20240206 in the sref's grid2obs driver script and run
it.
This can save time.

  For HREF, all plot files are good. All precip stats are good. The
grid2obs job is still running and so far so good.

Binbin


On Fri, Feb 9, 2024 at 2:58 PM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Yes, NARRE's stat and plot output files are all good
>
> Binbin
>
> On Fri, Feb 9, 2024 at 1:50 PM Shelley Melchior ***@***.***>
> wrote:
>
>> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , while href
>> finished up, I went ahead and started narre.
>> NARRE
>>
>> I have tested all of the narre stats and plots processes.
>> logfiles: /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_*.o*
>> COMOUT=
>> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240208
>> COMOUTplots=
>> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240208
>> I see no cpreqs and I see many cp -vs. 👍
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1936438425>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBMPFPI3FMQJR5T7MTMVC3YSZVVZAVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZWGQZTQNBSGU>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Not for sure if you have re-run the sref precip job. Its o file shows it is
still the output of the old version (before updated gather script).

Binbin

On Fri, Feb 9, 2024 at 3:16 PM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Shelley,
>
>   I found the problem: the VIS and HGT (ceiling) are missing in the final
> stat grid2obs file.
> Further debugging: you ram sref grid2obs for VDATE=20240207 only. But it
> depends on grid2obs_cnv.
> So you should run grid2obs_cnv first. After it is finished, run grid2obs.
> Another option: I noticed that you have run  sref grid2obs cnv for
> VDATE=20240206. Its output data are
> saved in the COMOUTsmall (e.g. atmos.VDATE directory)
> So you can set VDATE=20240206 in the sref's grid2obs driver script and run
> it.
> This can save time.
>
>   For HREF, all plot files are good. All precip stats are good. The
> grid2obs job is still running and so far so good.
>
> Binbin
>
>
> On Fri, Feb 9, 2024 at 2:58 PM Binbin Zhou - NOAA Affiliate <
> ***@***.***> wrote:
>
>> Yes, NARRE's stat and plot output files are all good
>>
>> Binbin
>>
>> On Fri, Feb 9, 2024 at 1:50 PM Shelley Melchior ***@***.***>
>> wrote:
>>
>>> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , while href
>>> finished up, I went ahead and started narre.
>>> NARRE
>>>
>>> I have tested all of the narre stats and plots processes.
>>> logfiles: /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_*.o*
>>> COMOUT=
>>> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240208
>>> COMOUTplots=
>>> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240208
>>> I see no cpreqs and I see many cp -vs. 👍
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1936438425>, or
>>> unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AQBMPFPI3FMQJR5T7MTMVC3YSZVVZAVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZWGQZTQNBSGU>
>>> .
>>> You are receiving this because you were mentioned.Message ID:
>>> ***@***.***>
>>>
>>
>>
>> --
>>
>> Binbin Zhou
>>
>> Physical Scientist
>>
>> Lynker at NOAA/NWS/NCEP/EMC
>>
>> 5830 University Research Ct.
>>
>> College Park, MD 20740
>>
>> ***@***.***
>>
>> 301-683-3683
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks Binbin! I will re-run sref precip stats.
I am satisfied with the sref grid2obs stats explanation and do not think that the re-run is necessary.

Thanks for checking narre and the rest of href that has completed so far.

ShelleyMelchior-NOAA commented:
Hi @BinbinZhou-NOAA , href g2o stats has completed. 
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_grid2obs_stats.o132341528`
I see no `cpreq`s and I see `cp -v`s. 👍 

The final stats files match what is in para!
```
[dlogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ll /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/cam/href.20240208
total 229016
-rw-r--r-- 1 shelley.melchior vpppg 112171700 Feb  9 19:37 evs.stats.href.grid2obs.v20240208.stat
-rw-r--r-- 1 shelley.melchior vpppg  57611453 Feb  9 19:55 evs.stats.href_mean.grid2obs.v20240208.stat
-rw-r--r-- 1 shelley.melchior vpppg    153264 Feb  9 17:15 evs.stats.href_mean.spcoutlook.v20240208.stat
-rw-r--r-- 1 shelley.melchior vpppg  64568633 Feb  9 20:29 evs.stats.href_prob.grid2obs.v20240208.stat
[dlogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ll /lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0/stats/cam/href.20240208
total 229016
-rw-r--r-- 1 emc.vpppg vpppg 112171700 Feb  9 11:19 evs.stats.href.grid2obs.v20240208.stat
-rw-r--r-- 1 emc.vpppg vpppg  57611453 Feb  9 11:38 evs.stats.href_mean.grid2obs.v20240208.stat
-rw-r--r-- 1 emc.vpppg vpppg    153264 Feb  9 09:01 evs.stats.href_mean.spcoutlook.v20240208.stat
-rw-r--r-- 1 emc.vpppg vpppg  64568633 Feb  9 12:11 evs.stats.href_prob.grid2obs.v20240208.stat
```

BinbinZhou-NOAA commented:
Shelley,

  Yes, now all of href output files are good.

Binbin


On Fri, Feb 9, 2024 at 3:40 PM Shelley Melchior ***@***.***>
wrote:

> Hi @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , href g2o stats
> has completed.
> logfile:
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_grid2obs_stats.o132341528
> I see no cpreqs and I see cp -vs. 👍
>
> The final stats files match what is in para!
>
> [dlogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ll /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/cam/href.20240208
> total 229016
> -rw-r--r-- 1 shelley.melchior vpppg 112171700 Feb  9 19:37 evs.stats.href.grid2obs.v20240208.stat
> -rw-r--r-- 1 shelley.melchior vpppg  57611453 Feb  9 19:55 evs.stats.href_mean.grid2obs.v20240208.stat
> -rw-r--r-- 1 shelley.melchior vpppg    153264 Feb  9 17:15 evs.stats.href_mean.spcoutlook.v20240208.stat
> -rw-r--r-- 1 shelley.melchior vpppg  64568633 Feb  9 20:29 evs.stats.href_prob.grid2obs.v20240208.stat
> [dlogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ll /lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0/stats/cam/href.20240208
> total 229016
> -rw-r--r-- 1 emc.vpppg vpppg 112171700 Feb  9 11:19 evs.stats.href.grid2obs.v20240208.stat
> -rw-r--r-- 1 emc.vpppg vpppg  57611453 Feb  9 11:38 evs.stats.href_mean.grid2obs.v20240208.stat
> -rw-r--r-- 1 emc.vpppg vpppg    153264 Feb  9 09:01 evs.stats.href_mean.spcoutlook.v20240208.stat
> -rw-r--r-- 1 emc.vpppg vpppg  64568633 Feb  9 12:11 evs.stats.href_prob.grid2obs.v20240208.stat
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1936573958>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFKTAGM7SJC3EOHJAMLYS2CTXAVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZWGU3TGOJVHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA committed:
Adding item 86, sub-item 38 into this PR.

ShelleyMelchior-NOAA committed:
t stash apply
Merge branch 'feature/reg_ensemble_fix_cpreq' of https://github.com/BinbinZhou-NOAA/EVS into feature/reg_ensemble_fix_cpreq

ShelleyMelchior-NOAA commented:
@BinbinZhou-NOAA , I snuck another changed file into your PR. It is to cover Item 86, sub-item 38 on Wei's document. Just FYI to please run a `git pull` in your fork.

BinbinZhou-NOAA commented:
Shelley,

  Sure, thanks!

Binbin

On Fri, Feb 9, 2024 at 4:20 PM Shelley Melchior ***@***.***>
wrote:

> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , I snuck another
> changed file into your PR. It is to cover Item 86, sub-item 38 on Wei's
> document. Just FYI to please run a git pull in your fork.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1936618180>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFMJAINGSXME6BK76Z3YS2HH7AVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZWGYYTQMJYGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA committed:
Including a big fix for EVS AQM that was indentified.

ShelleyMelchior-NOAA commented:
HI @BinbinZhou-NOAA , the sref precip stats concluded and the final stat file is in COMOUT! 🎉 It also compares well w/ the parallel.
```
[dlogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ll /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/mesoscale/sref.20240206
total 2968
drwxr-sr-x 2 shelley.melchior vpppg    4096 Feb  9 16:25 apcp24mean
-rw-r--r-- 1 shelley.melchior vpppg 3031873 Feb  9 22:05 evs.stats.sref.precip.v20240206.stat
[dlogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ll /lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0/stats/mesoscale/sref.20240206
total 8176
drwxr-sr-x 2 emc.vpppg vpppg    4096 Feb  8 10:56 apcp24mean
-rw-r--r-- 1 emc.vpppg vpppg 5330307 Feb  8 10:31 evs.stats.sref.grid2obs.v20240206.stat
-rw-r--r-- 1 emc.vpppg vpppg 3031879 Feb  8 11:09 evs.stats.sref.precip.v20240206.stat
```

I think this PR is in good shape to merge. ✔️

ShelleyMelchior-NOAA commented:
Hi @BinbinZhou-NOAA , once you confirm your support to merge this PR, please also run a sync fork.

BinbinZhou-NOAA commented:
Shelley,

  Yes, the sref precip stat file is good. Now all regional ensemble stat
and plot files are good.

Thanks!

Binbn

On Fri, Feb 9, 2024 at 5:11 PM Shelley Melchior ***@***.***>
wrote:

> HI @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , the sref
> precip stats concluded and the final stat file is in COMOUT! 🎉 It also
> compares well w/ the parallel.
>
> [dlogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ll /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/mesoscale/sref.20240206
> total 2968
> drwxr-sr-x 2 shelley.melchior vpppg    4096 Feb  9 16:25 apcp24mean
> -rw-r--r-- 1 shelley.melchior vpppg 3031873 Feb  9 22:05 evs.stats.sref.precip.v20240206.stat
> [dlogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ll /lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0/stats/mesoscale/sref.20240206
> total 8176
> drwxr-sr-x 2 emc.vpppg vpppg    4096 Feb  8 10:56 apcp24mean
> -rw-r--r-- 1 emc.vpppg vpppg 5330307 Feb  8 10:31 evs.stats.sref.grid2obs.v20240206.stat
> -rw-r--r-- 1 emc.vpppg vpppg 3031879 Feb  8 11:09 evs.stats.sref.precip.v20240206.stat
>
> I think this PR is in good shape to merge. ✔️
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/433#issuecomment-1936671330>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLMI7NZIOTXRTKZ6ATYS2NH3AVCNFSM6AAAAABC24CRTCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZWGY3TCMZTGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/reg_ensemble_fix_cpreq

ShelleyMelchior-NOAA reviewed (approved):
Tests yield expected results. This PR is approved for merge.
